cpanfile

```diff
@@ -49,7 +49,7 @@ requires 'JSON::Validator';
 requires 'JavaScript::Minifier::XS', '>= 0.11';
 requires 'LWP::Protocol::https';
 requires 'LWP::UserAgent';
-requires 'Minion', '>= 10.12';
+requires 'Minion', '>= 10.22';
```

Comment:
It would actually be just a test requirement but I guess distinguishing this won't bring us an advantage in practice.

Reply:
Probably counter productive since we might accidentally end up depending on newer features. And then we wouldn't notice it since we deploy and test with the latest version only. It would only fail for some users who happen to use older versions.